docs/start/quickstart.md

```diff
@@ -162,7 +162,7 @@ const app = express();
 app.use(express.static("build/client"));
 
 // and your app is "just a request handler"
-app.all("*", createRequestHandler({ build }));
+app.all(/(.*)/, createRequestHandler({ build }));
```

Comment:
Are these funcitonally equivalent?  If so, could we use this verison instead and avoid the regex?
```suggestion
app.all("/{*splat}", createRequestHandler({ build }));
```

Reply:
@brophdawg11 
Yes, also confirmed `"/{*splat}"` works fine.